### ISSUE TITLE
voyager crashes and needs to be restarted

### ISSUE THREAD

**Issue by jbibla** [cosmos/voyager#1279]
`v0.10.1 macOS`

not sure why this happens. it is a reoccurring issue for me. i would suggest we add a button on the modal for rebooting voyager if this happens - otherwise the user has to manually exit and reopen the app.

<img width="907" alt="screen shot 2018-09-04 at 10 54 51 am" src="https://user-images.githubusercontent.com/6021933/45039160-fb8c6100-b030-11e8-9928-d072cb11e70d.png">

```
Thu Aug 30 2018 14:22:38 GMT-0400 (EDT) Running Cosmos-UI

...

main-process: gaiacli: I[09-04|14:52:30.606] Served RPC HTTP response                     module=rest-server method=GET url=/node_version status=200 duration=269 remoteAddr=127.0.0.1:55597

main-process: Node 45.77.53.208:26657 uses SDK version 0.23.0 which is incompatible to the version used in Voyager 0.24.1
main-process: pinging node: http://10.140.0.2:26657
main-process: gaiacli: I[09-04|14:52:31.605] Served RPC HTTP response                     module=rest-server method=GET url=/validatorsets/latest status=200 duration=321 remoteAddr=127.0.0.1:55009

main-process: gaiacli: I[09-04|14:52:32.193] Served RPC HTTP response                     module=rest-server method=GET url=/validatorsets/latest status=200 duration=389 remoteAddr=127.0.0.1:55009

main-process: Node 10.140.0.2 is down
main-process: pinging node: http://5.135.2.83:26657
main-process: Node 5.135.2.83 is alive
main-process: Picked node: 5.135.2.83
main-process: starting gaia rest server with nodeIP 5.135.2.83:26657
main-process: startLCD /Users/jordan/.cosmos-voyager/gaia-8001/lcd
main-process: spawning /private/var/folders/mw/jjgtf89s2yqby0vqjd15x1d80000gn/T/AppTranslocation/143EF26D-E14A-416C-9977-54D2090FC2EE/d/Cosmos Voyager.app/Contents/Resources/app/bin/gaiacli with args ""advanced" "rest-server" "--laddr" "tcp://localhost:9071" "--home" "/Users/jordan/.cosmos-voyager/gaia-8001/lcd" "--node" "5.135.2.83:26657" "--chain-id" "gaia-8001""
main-process: gaiacli: I[09-04|14:52:34.018] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:55620

main-process: gaia rest server ready
main-process: Signaling connected node
main-process: gaiacli: panic: resource temporarily unavailable

goroutine 1 [running]:
github.com/cosmos/cosmos-sdk/client/lcd.createHandler(0xc42012bf10, 0x5, 0x7ffeefbffcbc)
	/go/src/github.com/cosmos/cosmos-sdk/client/lcd/root.go:78 +0x634
github.com/cosmos/cosmos-sdk/client/lcd.ServeCommand.func1(0xc4200afb00, 0xc420488f00, 0x0, 0x8, 0x0, 0x0)
	/go/src/github.com/cosmos/cosmos-sdk/client/lcd/root.go:40 +0x8e
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).execute(0xc4200afb00, 0xc420488e00, 0x8, 0x8, 0xc4200afb00, 0xc420488e00)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:698 +0x46d
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x20dd820, 0x1a163f9, 0x24, 0xc420472688)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).Execute(0x20dd820, 0xc4201916c0, 0x20dd900)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:736 +0x2b
github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tendermint/libs/cli.Executor.Execute(0x20dd820, 0x1b5ffe8, 0x2, 0xc420ac2fc0)
	/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tendermint/libs/cli/setup.go:89 +0x4e
main.main()
	/go/src/github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaiacli/main.go:151 +0xcce

main-process: gaiacli exited with code 2
main-process: The gaiacli rest-server (LCD) exited unplanned
main-process: gaiacli: I[09-04|14:52:34.079] Served RPC HTTP response                     module=rest-server method=GET url=/keys status=200 duration=0 remoteAddr=127.0.0.1:55009

main-process: gaiacli: I[09-04|14:52:34.169] Served RPC HTTP response                     module=rest-server method=GET url=/node_version status=200 duration=146 remoteAddr=127.0.0.1:55621

main-process: Node 5.135.2.83:26657 uses SDK version 0.23.0 which is incompatible to the version used in Voyager 0.24.1
main-process: pinging node: http://13.59.160.164:26657
main-process: gaiacli: I[09-04|14:52:35.010] Served RPC HTTP response                     module=rest-server method=GET url=/validatorsets/latest status=200 duration=390 remoteAddr=127.0.0.1:55009

main-process: Node 13.59.160.164 is down
main-process: pinging node: http://172.16.163.101:26657
main-process: Node 172.16.163.101 is down
main-process: pinging node: http://18.130.191.207:26657
main-process: gaiacli: I[09-04|14:52:40.596] Served RPC HTTP response                     module=rest-server method=GET url=/validatorsets/latest status=200 duration=526 remoteAddr=127.0.0.1:55009
```

**Comment by NodeGuy** [comment]
Please label all new issues with `discuss` until your new process is approved.

**Comment by NodeGuy** [comment]
What are the steps necessary to recreate this?

**Comment by jbibla** [comment]
1. open voyager 👐 
2. log in 🔑 
3. go for a canoe 🛶 

basically, if you leave the app open for a while (20-1hr - not sure) - it will crash eventually. can't tell if it's _voyager_ or an issue with the node voyager is connected to.

**Comment by faboweb** [comment]
This probably has to do with an old gaia process not being terminated properly when reconnecting. The gaia process would then lock the keys db. The next gaia process will find the key db locked and crashes.

**Comment by faboweb** [comment]
Fixed by #1314
